test/conformance/testing/include/uur/checks.h

```diff
@@ -8,7 +8,7 @@
 
 #include <gtest/gtest.h>
 #include <ur_api.h>
-#include <ur_params.hpp>
+#include <ur_print.hpp>
```

Comment:
I think we want this to be part of the ur api implemented by the loader. So should be in the `/include` folder.

Reply:
The C++ header is already in the `include/` dir. Did you mean that the C header along with the implementation should be in the `include/`?

Reply:
yes.